CHANGELOG.md

```diff
@@ -1,14 +1,26 @@
 # nf-core/hic: Changelog
 
+## v1.1.0
+
+* Support 'N' base motif in restriction/ligation sites
+* Support multiple restriction enzymes/ligattion sites (comma separated) (#31)
```

Comment:
This is automatically done by github I think ?

Reply:
Sorry, at a training course so review was done with a bit of haste 😅 

I dont think it is because you wont be able to click the link here:
https://github.com/nf-core/hic/blob/dev/CHANGELOG.md

Reply:
ok done